runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkKeyedCombineFn.java

```diff
@@ -40,14 +40,14 @@
 
 
 /**
- * A {@link org.apache.beam.sdk.transforms.CombineFnBase.PerKeyCombineFn}
+ * A {@link org.apache.beam.sdk.transforms.CombineFnBase.GlobalCombineFn}
  * with a {@link org.apache.beam.sdk.transforms.CombineWithContext.Context} for the SparkRunner.
  */
 public class SparkKeyedCombineFn<K, InputT, AccumT, OutputT> extends SparkAbstractCombineFn {
```

Comment:
In fact the code paths are different enough that we should address this separately. It is easy, but not "just a rename", mostly due to keeping the WindowedValue wrappers in place everywhere. It is definitely a runner implementation detail that it chooses to specialize.